app/app_test.go

```diff
@@ -0,0 +1,12 @@
+package app
+
+import (
```

Comment:
Move it to another place, or remove it if it is only for debug.

Reply:
fixed